controller/DeliveryMgmt.php

```diff
@@ -114,30 +112,13 @@ public function wizard()
     public function getAvailableLtiProviders()
```

Comment:
Please add missing tests

Reply:
Missing tests are pretty hard here with lots of static calls, `new` statements, altered states instead of returned results... 
This is the reason why I didn't make the missing tests beforehand.
Would you agree on postponing these, please? I don't want to spend the rest of the sprint only for those...

Reply:
We can create an additional task for the missing tests in this case.

Reply:
Yeah, let's keep them for later, if you don't mind. Even the one function I modified is fetching parameters from a PsrRequest and it's outcome is a task creation... :-(